docs/API.md

```diff
@@ -304,6 +291,14 @@ I.e. a string matching `/^[a-zA-Z0-9-.@]{1,50}$/`.
 
 Represents the name of an authenticated account (see *[Privilege levels]*)
 
+### Validation error data
+
+Basic data type: array
+
+The items of the array are objects with two keys, `"path"` and `"message"`:
+* `"path"`: a string. The JSON path a element in the request's param object.
```

Comment:
Is the way of giving paths standardized in JSON? Is the a reference? Could we have an example in the text?

Reply:
It is a standard, probably not the only one but I had the name wrong, [RFC6901](https://datatracker.ietf.org/doc/html/rfc6901) standardized it as JSON Pointer. I will update the api doc accordingly.